pkgs/by-name/ve/vesktop/package.nix

```
@@ -0,0 +1,62 @@
+{ buildNpmPackage, fetchFromGitHub, vencord, electron, pipewire, lib }:
```

nixpkgs-fmt?

---

nixpkgs-fmt doesn't reformat anything here

---

hmm. I would expect it to do something like this:
```suggestion
{ lib
, buildNpmPackage
, fetchFromGitHub
, vencord
, electron
, pipewire
}:
```